Habitica/res/layout/widget_todo_list_row.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/widget_list_row"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/white">
+
+        <FrameLayout
+            android:id="@+id/checkbox_background"
+            android:layout_width="@dimen/button_compact_width"
+            android:layout_height="match_parent"
+            android:layout_gravity="center">
+
+            <ImageView
+                android:layout_width="@dimen/checkbox_compact_size"
+                android:layout_height="@dimen/checkbox_compact_size"
+                android:layout_gravity="center"
+                android:src="@drawable/todo_unchecked" />
+        </FrameLayout>
+
+        <TextView
+            android:id="@+id/dailies_text"
```

Comment:
"todo_text" would be my preference for consistency with the wiki and Socialite posts, and some of the website's code. We use "task" as a generic term for Habits and Dailies and To-Dos (also even Rewards in some programming contexts).